ui/ui-components/pages/dashboard.tsx

```diff
@@ -52,3 +54,10 @@ export default function Page(props) {
     </>
   );
 }
+
+// this getServerSideProps doesn't actually return anything, but we to want to make a request to see if the user is signed in
```

Comment:
I feel like we should make this call in `_app`

Reply:
Making this call in _app would trigger a redirect, no?

Reply:
A few problems I see:
- not every page is authenticated, so then _app would need to be aware of which pages should have this call or not, or at least how to handle the error differently on each page
- this is an extra request on every page, since most (all but 2) pages have some hydration step of their own

Reply:
Maybe we should try to do what Cloud does, and create a `useAuthenticatedPage` hook?
That way redirects are triggered at the React level rather than the request level, and we can just authenticate in `_app` to know if we're logged in or not.

Reply:
Perhpas! I think that's a new story though.

Reply:
Are there other pages that need something like this so the fail properly?

Reply:
AFAIK this is the only page that didn't make a `getInitialProps` or `getServerSideProps`